File: Logic/3 - JavaScript/exercicio5.js
var valor, desconto;
    valor = prompt("Qual o valor do produto?");
    desconto = prompt("Qual será o desconto aplicado?");
    valor=valor*(1-(desconto/100));
    if(valor >= 500)
    {
        alert("O produto custará R$" + valor + " com " + desconto + "% de desconto.\nO produto está CARO");
        
    }
    else
    {
        alert("O produto custará R$" + valor + " com " + desconto + "% de desconto.\nO produto está BARATO");
    }

File: Exercicios e atividades/Exercicios/1 - C++/exercicio5.cpp
#include <stdio.h>
int main(void)
{
    float valor, desconto;
    printf("Qual o valor do produto?");
    scanf("%f",&valor);
    printf("Qual sera o desconto aplicado?");
    scanf("%f", &desconto);
    valor=valor*(1-(desconto/100));
    if (valor >= 500)
    {
        printf("O produto custara R$%f com o desconto.\n\nO produto esta CARO", valor);
    }
    else
    {
        printf("O produto custara R$%f com o desconto.\n\nO produto esta BARATO", valor);
    }
}
